xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-webjar/src/main/webjar/lightbox.js

```diff
@@ -172,36 +172,39 @@ define('xwiki-lightbox', [
   /*
    * Make sure that the toolbar will remain open also while hovering it, not just the image.
    */
-  var keepToolbarOpenOnHover = function(image, timeout) {
-    timeout = setTimeout(function() {
-      image.popover('hide');
+  var keepToolbarOpenOnHover = function(image, hideTimeout) {
+    hideTimeout = setTimeout(function() {
```

Comment:
You get ``hideTimeout`` as parameter and then you overwrite it right away. Doesn't make sense. It means you don't need it as input parameter. Note that this doesn't overwrite the ``hideTimeout`` from ``enableToolbarPopovers``. So something is fishy here.

Also, a comment wouldn't hurt:

```suggestion
    // Hide the image popover after 3 seconds (if the mouse doesn't enter the popover).
    hideTimeout = setTimeout(function() {
```